lib/environment.ts

```diff
@@ -20,17 +20,18 @@ export const setAuthToken = async (token: string) => {
   console.log("Setting authToken: ", maskToken(token));
 
   if (process.env.VERCEL === "1") {
-    await fetch(
-      process.env.SALEOR_MARKETPLACE_REGISTER_URL as string,
-      {
-        method: "POST",
-        headers: { "Content-Type": "application/json" },
-        body: JSON.stringify({
-          auth_token: token,
-          marketplace_token: process.env.SALEOR_MARKETPLACE_TOKEN,
-        }),
-      },
+    invariant(
+      process.env.SALEOR_MARKETPLACE_REGISTER_URL,
+      "Env var SALEOR_MARKETPLACE_REGISTER_URL is not configured."
     );
+    await fetch(process.env.SALEOR_MARKETPLACE_REGISTER_URL as string, {
```

Comment:
Since you're asserting that `SALEOR_MARKETPLACE_REGISTER_URL` was set on lines 23-25, can we drop `as string` here?
```suggestion
    await fetch(process.env.SALEOR_MARKETPLACE_REGISTER_URL, {
```